apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.ts

```diff
@@ -326,7 +326,6 @@ router.get(
 
       const cursor = await sqldb.queryCursor(sql.assessment_instance_files, {
         assessment_id: res.locals.assessment.id,
-        limit: 100,
```

Comment:
Was this just unused?

Reply:
Yup. And since there is no test that calls this code, and since production doesn't trigger an error for unused params, this was left inadvertently in.